examples/hello_LoRa/hello_LoRa.ino

```diff
@@ -25,6 +25,33 @@ uint8_t AppSkey[16] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x
 uint8_t DevAddr[4] = { 0x00, 0x00, 0x00, 0x00 };
 
 /************************** Example Begins Here ***********************************/
+// Start TxPower settings //
+uint8_t const TxPower = 0xFF; // default value. Don't modify this value [default: 0xFF]
+
+// uncomment this section if you want to control the TxPower, and comment the above line.
+/*
+bool PaBoost = 1; // 8th bit [default: 1]
+uint8_t OutputPower = 1; // 1-4 bit (DEC values: 0-15) [default: 15] +2 to +17dBm.
+uint8_t MaxPower = 4; // 5-7 bit, (DEC values: 0-7) [default: 4] BUG? Range -4 to 0 but DOC says -4 to +15dBm
+
+// function to pack the data for TXpower.
+uint8_t packDataPower(){
```

Comment:
I think all calculation should be inside the lora.setPower(TxPower) function.

Reply:
It seems the setPower (aka the function in .ino) is not ready yet, although I tested and it works. I managed to get -100 to -9 RSSI from 30 cm distance. I think some details are missing. I am getting some ideas from lmic. https://github.com/mcci-catena/arduino-lmic/commit/49b495def976212b954b0af3904ae492d8f468da

Is preferable to commit in this request, or start new?

Reply:
@clavisound Could you make a new PR specifically for `setPower` when you're ready?

Reply:
Yes, I had the same idea. I came to write you.

I will submit for frame port only. I will configure the default port to 1, so the examples will not brake and I will try to write separate example for frame port.

You can cancel those commits.

Thank you both for the time.

Reply:
Sounds good, closing this PR and I'm excited to test your next PR.